docs/using-kontena/loadbalancer.md

```diff
@@ -183,6 +183,40 @@ loadbalancer:
       type: env
 ```
 
+## Canary deployment
+
+[Canary deployments]() is a great way to break new service releases into production by moving traffic incrementally to new release of a given service. Kontena loadbalancer supports this by aliasing two services to use same configuration on loadbalancer level.
+```
+app_version1:
+  image: my-app:1.0
+  environment:
+    - KONTENA_LB_MODE=http
+    - KONTENA_LB_BALANCE=roundrobin
+    - KONTENA_LB_INTERNAL_PORT=80
+    - KONTENA_LB_VIRTUAL_HOSTS=my-app.example.com
+    - KONTENA_LB_SERVICE_ALIAS=my-app
+  links:
+    - internet_lb
+```
+
+Now when you are ready to deploy a new version of your app you can deploy that using following configuration:
+```
+app_version2:
+  image: my-app:2.0
+  environment:
+    - KONTENA_LB_MODE=http
+    - KONTENA_LB_BALANCE=roundrobin
+    - KONTENA_LB_INTERNAL_PORT=80
+    - KONTENA_LB_VIRTUAL_HOSTS=my-app.example.com
+    - KONTENA_LB_SERVICE_ALIAS=my-app
```

Comment:
This alias is global across all stacks, maybe it would be a good idea to use `KONTENA_LB_SERVICE_ALIAS=$STACK/my-app` as the example?